### ISSUE TITLE
unserialize() expects parameter 1 to be string

### ISSUE THREAD

**Issue by rain2o** [bentideswell/magento2-wordpress-integration#59]
I am receiving the following exception when loading a page that uses the NavMenu widget. Looks like it started happening after upgrading to the latest version (1.2.0.10) of the main integration extension. 

`Warning: unserialize() expects parameter 1 to be string, array given in /Users/jrainwater/code/patnash2.dev/app/code/FishPig/WordPress/Model/Menu/Item.php on line 243`

It is happening in the function `getCssClass` when attempting to get the `_menu_item_classes` value. It appears to be returning an array instead of a string. That value is stored in the database in this form `a:1:{i:0;s:12:"my-class";}`, and is returned from `Model/Meta/AbstractMeta.php::getMetaValue()` as `array (0 => 'my-class');`

Basically, it looks like the data is getting unserialized twice.

**Environment:**
Magento - 2.1.8 CE
FishPig_WordPress - 1.2.0.10
FishPig_WordPress_ACF - 1.0.0.23
FishPig_WordPress_Root - 1.0.0.3
WordPress - 4.8.2
Advanced Custom Fields PRO - 5.6.3

**Comment by bentideswell** [comment]
Thanks for the detailed breakdown of the issue. I can't replicate the issue but I have added a check to see whether the value is an array and if it is, it skips the unserialize call. If you upgrade, the issue should be resolved.

I will close this issue but if upgrading doesn't solve the problem then please re-open it.

**Comment by rain2o** [comment]
Thanks, that did the trick. 

I think it was due to the WordPress_ACF extension. The debugger took me into it, but as it's encoded I wasn't able to go any further. I'm guessing maybe it was getting unserialized in there somewhere as well.
